src/main/java/org/zalando/nakadi/Application.java

```diff
@@ -2,8 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
 
 @SpringBootApplication
+@ComponentScan(basePackages = {"org.zalando.nakadi", "org.zalando.nakadi.config"})
```

Comment:
Could explain why it requires component scan with the new version of spring boot ?